src/runtime/components/LitWrapperServer.ts

```diff
@@ -107,11 +107,14 @@ export default defineComponent({
   },
 
   render() {
-    const attrs = this.getAttributesToRender();
+    // This is the case when the node is a fragment created by a v-for on a lit element
+    if (typeof this.litElementTagName === "symbol") {
+      return h(this.litElementVnode);
+    }
 
     return h(this.litElementTagName, {
       innerHTML: this.litSsrHtml,
-      ...attrs
+      ...this.litElementVnode.props
```

Comment:
Will this not create a problem again related to #30 ?

Reply:
@steveworkman what about this one?

Reply:
`props` are all the attributes/properties that were passed to the lit element, regardless as to whether they should definitely be reflected or not. This seems to resolve the issues we were seeing with the accordion and gives an HTML rendering as the author intended it. I don't get any hydration issues on any of the components that we had in the suite before I added the step indicator